### PR TITLE
Roundstart Plate Carriers

### PR DESCRIPTION
**Pull request by Geevies**
* Security Officers and the Head of Security now spawn with their plate carrier, but the armoured plate does not come with it attached, and must be attached in the locker room. This can be customized with accessories.
* Removing accessories with the radial accessory removal menu will now reopen that menu until either all the accessories are removed, or you click the close button.
* Plate Carrier armor accessories will now always render below other accessories, so you don't need to shuffle everything to layer them correctly.

https://github.com/Aurorastation/Aurora.3/assets/22774890/5f86f96a-8b40-4b61-81d6-e8521531452f

### CODE REVIEW

**Comment by Geevies** [comment]
!review

**Comment by Geevies** [comment]
!review

**Review by NonQueueingMatt — APPROVED** [review]

**Comment by DreamySkrell** [comment]
image
...why? is this intentional?

File: code/game/objects/structures/crates_lockers/closets/secure/security.dm [hunk 1]
```diff
@@ -240,35 +250,52 @@
...
 	new /obj/item/storage/belt/security/full(src)
 	new /obj/item/clothing/suit/storage/hazardvest/security/officer(src)
 
+// this is the locker that spawns in the locker room
+/obj/structure/closet/secure_closet/security/proc/spawn_armor()
+	new /obj/item/clothing/accessory/armor_plate(src)
+
+// this is the locker that spawns in the security checkpoints around the ship
+/obj/structure/closet/secure_closet/security/spare/spawn_armor()
+	new /obj/item/clothing/suit/armor/carrier/officer(src)
```
Comment (DreamySkrell) [inline code/game/objects/structures/crates_lockers/closets/secure/security.dm:297]: really, *really* not a fan of this approach with this proc and `/spare` locker

Comment (DreamySkrell) [reply code/game/objects/structures/crates_lockers/closets/secure/security.dm:297]: also if sec officer lockers start with only carriers and no vests, you can't quickly give armor to non-sec or even just cadets (and have to run to a checkpoint instead)

imo sec lockers should still have vest (as in both vests and plates, just separate)

**Comment by NonQueueingMatt** [comment]
I've thought about it for a bit and making carriers and plates spawn separate is not worth it imo.

**Pull request closed without merging**